### PR TITLE
Tests NewsPage

### PR DESCRIPTION
**Pull request by amyrchyk**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/NewsPage.java [hunk 1]
```diff
@@ -46,4 +52,16 @@ public NewsPage(WebDriver driver) {
         return news;
     }
     
+    public String getTitleNews() {
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/NewsPage.java:55]: rename if you return only first

Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/NewsPage.java]: done

File: src/test/java/com/qaprosoft/carina/demo/GsmArenaTests.java [hunk 2]
```diff
@@ -64,4 +71,40 @@ public void testVerifyIncorectPass() {
...
+	HomePage homePage = loginService.login(user.getEmail(), user.getPassword());
+	NewsPage newsPage = homePage.getFooterMenu().openNewsPage();
+	Assert.assertTrue(newsPage.isPageOpened(), "News page is not opened!");
+	final String searchQ = "iphone";
+	String resultSearch = "Results for \"" + searchQ + "\"";
+	List<NewsItem> news = newsPage.searchNews(searchQ);
+	Assert.assertEquals(newsPage.getTitleSearch(), resultSearch, "Titles aren't equals");
+	Assert.assertFalse(CollectionUtils.isEmpty(news), "News not found!");
+	for (NewsItem n : news) {
+	    System.out.println(n.readTitle());
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/GsmArenaTests.java:106]: better use Logger

Comment (amyrchyk) [reply src/test/java/com/qaprosoft/carina/demo/GsmArenaTests.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/components/LoginForm.java [hunk 3]
```diff
@@ -3,31 +3,34 @@
...
+    @FindBy(id = "nick-submit")
     public ExtendedWebElement logInButton;
-    
+
     public LoginForm(WebDriver driver, SearchContext searchContext) {
 	super(driver, searchContext);
     }
 
     public LoginPage login(String email, String pass) {
+	LOGGER.info("Email" + email);
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/components/LoginForm.java:29]: better
LOGGER.info("Type email - " + email);

File: src/main/java/com/qaprosoft/carina/demo/gui/components/LoginForm.java [hunk 4]
```diff
@@ -3,31 +3,34 @@
...
 	emailField.type(email);
+	LOGGER.info("Password" + pass);
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/components/LoginForm.java:31]: see above

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/ArticlePage.java [hunk 5]
```diff
@@ -2,20 +2,22 @@
...
     public ExtendedWebElement titleArticle;
-    
+
     public ArticlePage(WebDriver driver) {
 	super(driver);
     }
-    
+
     public String getTitleArticle() {
+	LOGGER.info("Title Article" + titleArticle.getText());
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/ArticlePage.java:20]: add space before  "

File: src/main/java/com/qaprosoft/carina/demo/gui/components/LoginForm.java (hunk 3 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/components/LoginForm.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/components/LoginForm.java (hunk 4 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/components/LoginForm.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/ArticlePage.java (hunk 5 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/ArticlePage.java]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**